public/static/locales/en/common.json

```diff
@@ -387,6 +387,7 @@
   "en": "en",
   "Enter answer": "Enter answer",
   "Accepted": "Accepted",
+  "Enter a valid establishment web address": "Enter a valid establishment web address",
```

Comment:
Move this to join the rest of the validation statements to keep them all together, and to match the en json file